### PR TITLE
Add documentation of new keys in /check_kyc

### PR DESCRIPTION
**Pull request by freemish**
Adds descriptions for new keys returned from the /check_kyc endpoint and lots of examples in the specs panel.

### CODE REVIEW

File: source/includes/_endpoints.md [hunk 1]
```diff
@@ -624,14 +753,43 @@ The handle for which KYC confirmation is being checked should be in the `header.
...
-| 400 | `"FAILURE"` | Bad request format - check `validation_details` for more information. |
-| 401 | `"FAILURE"` | `authsignature` or `usersignature` header was absent or incorrect. |
+There are several new keys in the /check_kyc response:
+
+* `entity_type`: Value will always be "individual" for individual users.
+* `valid_kyc_levels`: An array of KYC level names that can be requested for this user. All verifications with any of these KYC levels for the current user will be included in the verification history and used to compute the overall verification status.
+* `verification_status`: For individual users, this value can be one of: ["unverified", "pending", "passed", "failed", "review"]. There are descriptions of these statuses in the table below.
+* `verification_history`: The value at this key is an array of 0 or more verification objects. All objects in this array have the following keys:
+    * `verification_id`: A UUID identifying the verification run for this user.
```
Comment (johndburk) [inline source/includes/_endpoints.md:774]: Can the order of the verification_history keys in the main body's `Response` section align with the order of those same keys in the right panel?   I found myself continually searching through the Response section when referencing the example output.

File: source/includes/_endpoints.md [hunk 2]
```diff
@@ -547,10 +547,139 @@ Content-Type: application/json
...
+                "KYC Address Match"
+            ],
+            "score": 0.99,
+            "parent_verification": null
+        }
+    ],
+    "valid_kyc_levels": [
+        "DEFAULT"
+    ],
+    "status": "FAILURE"
```
Comment (johndburk) [inline source/includes/_endpoints.md:648]: Can the top-level key ordering be consistent for each response example?  Having them inconsistent makes it difficult to compare between the various response objects

**Comment by freemish** [comment]
Ooh, this is good feedback, will implement.

File: source/includes/_endpoints.md (hunk 1 above)
Comment (freemish) [reply source/includes/_endpoints.md:773]: should be resolved! but let me know if I screwed it up ;)

File: source/includes/_endpoints.md (hunk 2 above)
Comment (freemish) [reply source/includes/_endpoints.md]: same here, they *should* be consistent now if I didn't miss anything

File: source/includes/_endpoints.md [hunk 3]
```diff
@@ -547,10 +547,107 @@ Content-Type: application/json
...
+      "reasons": [],
+      "tags": [
+        "KYC Name Match",
+        "KYC SSN Match",
+        "KYC DOB Match",
+        "Phone Match",
+        "KYC Address Match"
+      ],
+      "score": 0.99,
+      "parent_verification": null
```
Comment (johndburk) [inline source/includes/_endpoints.md:571]: not seeing mention of this key in the docs on the left

Comment (freemish) [reply source/includes/_endpoints.md:571]: yeah, it's unused until KYB is usable by the outside world, so will always be null/unused until then... I'll make a note of it, though, sure

**Comment by freemish** [comment]
> why a draft? Looks good to go, not a WIP is it?

You're right :) I'm using the grayed-out "can't merge" to make sure I don't forget that this shouldn't be merged until these changes are actually released. Might be a better way to do that, but it's basically just a sanity check for me

**Pull request merged**